src/layouts/EditHomepage.jsx

```diff
@@ -985,6 +985,7 @@ const EditHomepage = ({ match }) => {
                             <Editable.EditableAccordionItem title="Hero section">
                               <HeroBody
                                 {...section.hero}
+                                highlights={section.hero.key_highlights || []}
```

Comment:
why is key_highlights in snake case?

Reply:
one of the beautiful treasures in our legacy code :)

i've opted not to fix this because it changes the shape and i want to keep this refactor scoped tightly